### ISSUE TITLE
FlexContainer renders as empty despite having button child.

### ISSUE THREAD

**Issue by neavilos** [citizenll/godot-flexbox#18]
Hi there,

I installed the Linux version of this plugin in Godot for 4.2. I'm not able to get anything to render to the screen when I use the FlexContainer. It just shows up as blank. I'm wondering if I could get some help figuring out why this is happening? My specific Godot build is v4.2.stable.mono.official [46dc27791] if that matters.

I had some difficulty setting up the plugin, especially with native class resolution for the Flexbox class, but I was able to resolve these the errors and confirmed it's registered now via gdscript. 

I shared some images below of the setup. I tried to play around with the flex template options to no avail. I've reset all the options back to their default settings except for size and position and anchoring.

![image](https://github.com/citizenll/godot-flexbox/assets/105667131/ae050d38-cd2a-454b-9707-789997917632)
![image](https://github.com/citizenll/godot-flexbox/assets/105667131/1437e9aa-8c57-49a9-8ee0-fae2601aba3c)
![image](https://github.com/citizenll/godot-flexbox/assets/105667131/568b7e12-b973-4dd6-b66f-9c146afddfbe)

**Comment by citizenll** [comment]
I can't replicate this issue on Windows, I'll test it later on an Ubuntu system when I have time.
![image](https://github.com/citizenll/godot-flexbox/assets/16896067/2c9c862d-e24c-4d3d-a078-d084d2b262d8)

**Comment by shuzhiwen** [comment]
This problem also occurs on MacOS

**Comment by neavilos** [comment]
Bumping to see if any progress has been made, especially since this seems to be a non-Windows issue.

**Comment by neavilos** [comment]
I have an update from my own research. It looks like the issue specifically belongs to the editor, not the runtime. I was able to display the 4 squares in the demo in plenty of configurations without issue, but I used a locally compiled binary of this repo's gdextension. I have not tested yet if it works with the pre-built version.

**Comment by citizenll** [comment]
Indeed, there is an issue with the precompiled files. I encountered the same problem in my Ubuntu system on VMware. I do not yet know why the files compiled by GitHub Actions cannot run. Additionally, on Mac, it seems that an extra version for Apple Silicon needs to be compiled.

**Comment by citizenll** [comment]
Fixed by commit 676258a: improved child minimum-size handling and layout application in FlexContainer.
